src/sort.rs

```diff
@@ -297,6 +308,14 @@ mod test {
         // println!("{}", sorted.to_string());
     }
 
+    #[test]
+    fn toml_workspace_deps_edit_check() {
+        let input = fs::read_to_string("examp/workspace_deps.toml").unwrap();
+        let expected = fs::read_to_string("examp/workspace_deps.sorted.toml").unwrap();
```

Comment:
I only added a `.sorted` one for this particular test to keep the scope of this PR small. I did a quick experiment by checking how easy it is to add them for the others but ran into unexpected differences between what I think it should be sorted to and what it actually does.

I decided that is not worth fixing as part of this but should rather be handled in another issue.

Reply:
W.r.t. shipping, please note that I'm not currently doing releases as the upgrade of the toml crate has lead to a couple regressions that I want fixed first (but I'm not very keen on investing time in myself). The details are in one of the most recently updated issues.